src/pretix/base/settings.py

```diff
@@ -2001,6 +2008,24 @@ def unserialize(cls, s):
             '_scheme': 'full_transcription',
         },
     }),
+    ('title_salutation_given_family', {
```

Comment:
Maybe we should call this salutation_title_given_family if that's the order? ;)

Reply:
woopsy yes!